payment_ippay_ach/static/src/css/ippay_ach.css

```diff
@@ -0,0 +1,11 @@
+/* Chrome, Safari, Edge, Opera */
+input#aba::-webkit-outer-spin-button,
+input#aba::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
+/* Firefox */
+#aba{
+  -moz-appearance:textfield;
+}
```

Comment:
I wonder if this helps / can help solve the dark backgroung/dark lfont color issue that the customer is seeing right now?